projects/pgai/tests/vectorizer/test_vectorizer_cli.py

```diff
@@ -151,19 +152,28 @@ def configured_ollama_vectorizer_id(
     source_table: str,
     cli_db: tuple[TestDatabase, Connection],
     test_params: tuple[int, int, int, str, str],
+    ollama_container: OllamaContainer,
 ) -> int:
     """Creates and configures an ollama vectorizer for testing"""
     _, concurrency, batch_size, chunking, formatting = test_params
     _, conn = cli_db
 
+    base_url = ollama_container.get_endpoint()
+    if "OLLAMA_HOST" in os.environ: 
+        # The user is running it's own Ollama container
+        base_url = os.environ["OLLAMA_HOST"]
+        ollama_container.stop()
```

Comment:
Could move this to the fixture I think. e.g. just call the fixture `ollama_url` and only start the container if the environment variable isn't set.

Reply:
You mean moving this logic to the `ollama_container` fixture?

Reply:
Fixed!